interface/VideoEncoderDefs.h

```diff
@@ -256,6 +256,7 @@ typedef struct VideoParamsCommon {
     uint32_t numRefFrames;
     VideoRateControl rcMode;
     VideoRateControlParams rcParams;
+    uint32_t referenceMode;
```

Comment:
before we fix it all, is it ok to create a VideoParamsVP9 just like VideoParamsAVC? If some one start use  VideoParamsCommon.referenceMode, it's hard to change it to VideoParamsVP9